includes/admin/class-admin.php

```diff
@@ -767,7 +767,7 @@ private function upgrade_bar() {
 
 			$modules = wpbdp()->modules;
 
-			if ( $modules->is_loaded( 'premium' ) || $modules->is_loaded( 'paypal' ) || $modules->is_loaded( 'googlemaps' ) ) {
+			if ( $modules->is_loaded( 'premium' ) || count( array_keys( $modules->get_modules() ) ) > 0 ) {
```

Comment:
Can we exclude the Enhanced Categories module from this check? A lot of people have gotten it free.